types/react-beautiful-dnd/index.d.ts

```diff
@@ -12,7 +12,7 @@ export type Id = string;
 export type DraggableId = Id;
 export type DroppableId = Id;
 export type TypeId = Id;
-export type ZIndex = number | string;
+export type ZIndex = React.CSSProperties['z-Index'];
```

Comment:
Should be `'zIndex'`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/c895c48502fcad26bb2caa00b0db0c603089f4b7/types/react/index.d.ts#L2438

Reply:
You are absolutely correct @kingdaro, my bad.